front/cypress/e2e/admin/internal_comments/idea_internal_comments.cy.ts

```diff
@@ -0,0 +1,120 @@
+    cy.get('.e2e-more-actions-list button').eq(1).contains('Edit');
+    cy.get('.e2e-more-actions-list button').eq(1).click();
+    cy.get('#e2e-internal-comment-edit-textarea')
+      .click()
+      .clear()
+      .type(editedComment);
+    cy.get('#e2e-save-internal-comment-edit-button').click();
+    cy.get('.e2e-parentcomment').contains(editedComment);
+
+    // Delete comment and check that comment is deleted
+    cy.get('[data-testid="moreOptionsButton"]').first().should('exist');
+    cy.get('[data-testid="moreOptionsButton"]').first().click();
+    cy.get('.e2e-more-actions-list button').eq(0).contains('Delete');
+    cy.get('.e2e-more-actions-list button').eq(0).click();
+    cy.get('#e2e-confirm-deletion').should('exist');
+    cy.get('#e2e-confirm-deletion').click();
+    cy.get('.e2e-parentcomment').should('not.exist');
+  });
+
+  it('allows users to reply to internal comments, edit and delete a reply', () => {
+    const internalComment = randomString();
+    const replyComment = randomString();
+    const editedReply = randomString();
+    cy.setAdminLoginCookie();
+    cy.visit(`admin/projects/${projectId}/ideas/${ideaId2}`);
+    cy.acceptCookies();
+    // Create comment and check that comment is created
+    cy.get('[data-cy="e2e-tab-internal"]').click();
+    cy.get('#submit-comment').should('exist');
+    cy.get('#submit-comment').click().type(internalComment);
+    cy.get('.e2e-submit-parentcomment').click();
+    cy.get('.e2e-parentcomment').contains(internalComment);
+
+    // Reply to comment and check that reply is created
+    cy.get('.e2e-comment-reply-button').should('exist');
+    cy.get('.e2e-comment-reply-button').click();
+    cy.get('#e2e-child-comment-text-area').should('exist');
+    cy.get('#e2e-child-comment-text-area').click().type(replyComment);
+    cy.get('.e2e-submit-childcomment').first().click();
+    cy.get('.e2e-childcomment').contains(replyComment);
+
+    // Edit reply and check that reply is edited
+    cy.get('[data-testid="moreOptionsButton"]').eq(1).should('exist');
+    cy.get('[data-testid="moreOptionsButton"]').eq(1).click();
+    cy.get('.e2e-more-actions-list button').eq(1).contains('Edit');
+    cy.get('.e2e-more-actions-list button').eq(1).click();
+    cy.get('#e2e-internal-comment-edit-textarea')
+      .click()
+      .clear()
+      .type(editedReply);
+    cy.get('#e2e-save-internal-comment-edit-button').click();
+    cy.get('.e2e-childcomment').contains(editedReply);
+
+    // Delete reply and check that reply is deleted
+    cy.get('[data-testid="moreOptionsButton"]').eq(1).should('exist');
+    cy.get('[data-testid="moreOptionsButton"]').eq(1).click();
+    cy.get('.e2e-more-actions-list button').eq(0).contains('Delete');
+    cy.get('.e2e-more-actions-list button').eq(0).click();
+    cy.get('#e2e-confirm-deletion').should('exist');
+    cy.get('#e2e-confirm-deletion').click();
+    cy.get('.e2e-childcomment').contains('This comment has been deleted.');
```

Comment:
```suggestion
    cy.get('.e2e-internal-child-comment').contains('This comment has been deleted.');
```

We could edit the class names too, to reduce chances we're testing in the wrong tab.